pkgs/applications/version-management/dvc/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, git
+, enableGoogle ? false
+, enableAWS ? false
+, enableAzure ? false
+, enableSSH ? false
+}:
+
+with python3Packages;
+buildPythonApplication rec {
+  pname = "dvc";
+  version = "0.24.0";
+
+  src = fetchFromGitHub {
+    owner = "iterative";
+    repo = "dvc";
+    rev = version;
+    sha256 = "163lnn8z4ig5dgl2jdkrlqwziyn28shswysq7gpamp8ri0g2ccx3";
+  };
+
+  buildInputs = [ git ];
```

Comment:
From https://github.com/iterative/dvc/issues/1509#issuecomment-457083344, it looks like this should be fixed in GitPython. I have pushed ee82616089ff920792b6c265068902c4fa3d4f7f to master.
```suggestion
```

Reply:
Done.